### ISSUE TITLE
Uncaught Error: Must pass a config parameter when constructing TextEditors

### ISSUE THREAD

**Issue by nmccready** [adrianlee44/atom-coffee-compile#39]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.2.0
**System**: Mac OS X 10.10.5
**Thrown From**: [coffee-compile](https://github.com/adrianlee44/atom-coffee-compile) package, v0.19.2

### Stack Trace

Uncaught Error: Must pass a config parameter when constructing TextEditors

```
At /Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:89

Error: Must pass a config parameter when constructing TextEditors
    at CoffeeCompileEditor.TextEditor [as constructor] (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:89:15)
    at new CoffeeCompileEditor (/Users/nem/.atom/packages/coffee-compile/lib/coffee-compile-editor.coffee:10:5)
    at /Users/nem/.atom/packages/coffee-compile/lib/main.coffee:47:18
    at Workspace.module.exports.Workspace.openURIInPane (/Applications/Atom.app/Contents/Resources/app.asar/src/workspace.js:454:22)
    at Object.module.exports.open (/Users/nem/.atom/packages/coffee-compile/lib/main.coffee:95:20)
    at Object.module.exports.display (/Users/nem/.atom/packages/coffee-compile/lib/main.coffee:69:6)
    at atom-workspace.atom.commands.add.coffee-compile:compile (/Users/nem/.atom/packages/coffee-compile/lib/main.coffee:21:71)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:533:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:356:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeydown (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:99:36)
    at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:3:61)
```

### Commands

```
     -0:04.4.0 coffee-compile:compile (atom-text-editor.editor.vim-mode.insert-mode.is-focused)
```

### Config

```json
{
  "core": {
    "themes": [
      "seti-ui",
      "monokai-dark"
    ],
    "destroyEmptyPanes": false,
    "disabledPackages": [
      "atom-color-highlight"
    ],
    "followSymlinks": false
  },
  "coffee-compile": {}
}
```

### Installed Packages

```coffee
# User
Stylus, v1.2.1
atom-jade, v0.3.0
autoclose-html, v0.19.0
autocomplete-paths, v1.0.2
coffee-compile, v0.19.2
color-picker, v2.0.13
dartlang, v0.4.14
git-diff-details, v0.21.0
highlight-column, v0.5.0
highlight-line, v0.11.0
highlight-selected, v0.10.1
language-markdown, v0.2.0
language-stylus, v0.2.0
line-diff-details, v1.1.3
linter, v1.11.1
linter-coffeelint, v1.1.2
linter-dartanalyzer, v0.3.2
linter-jade, v0.3.2
linter-jshint, v1.2.1
linter-sass-lint, v0.4.1
minimap, v4.16.0
minimap-git-diff, v4.1.8
minimap-pigments, v0.1.7
monokai-dark, v1.3.0
nuclide-diff-view, v0.0.32
open-recent, v4.0.1
pigments, v0.18.1
seti-ui, v0.8.1
vim-mode, v0.63.0
white-cursor, v0.6.0

# Dev
No dev packages
```

**Comment by adrianlee44** [comment]
Thanks for reporting this.

Looks like Atom 1.2 changed how `TextEditor` is initialized and I'm looking into fixing this.

**Comment by macpham** [comment]
:+1:

**Comment by adrianlee44** [comment]
This should be fixed with coffee-compile v0.20.0.

Thanks!

**Comment by nmccready** [comment]
Thank you all is well and my workflow is back to norm.
